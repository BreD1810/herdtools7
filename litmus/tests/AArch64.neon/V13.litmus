AArch64 V13
(* Tests LD2R, no offset *)
{
x=511;
0:X0=x;0:X1=y; 0:X2=z; 0:X3=a; 0:X4=b;
}

 P0                         ;
 LD2R {V0.8B,V1.8B}, [X0]   ;
 ST1 {V0.B}[0], [X1]        ;
 ST1 {V0.B}[7], [X2]        ;
 ST1 {V1.B}[0], [X3]        ;
 ST1 {V1.B}[7], [X4]        ;
 forall (y=255 /\ z=255 /\ a=1 /\ b=1)
