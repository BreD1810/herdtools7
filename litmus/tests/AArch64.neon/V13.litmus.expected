Test V13 Required
Histogram (1 states)
1000000:>a=1; b=1; y=255; z=255;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (y=255 /\ z=255 /\ a=1 /\ b=1) is validated
Hash=384aa5138aade24d5556f58ec2ffaf6e
Observation V13 Always 1000000 0
